GitCommitsForNonGitUsers.md

```diff
@@ -0,0 +1,42 @@
+#Git Commits For Non-Git Users
+
+This is a guide to aid others not familiar with git to contribute to our TIL repo!
+
+####FIRST TIME SETUP
+* copy clone to clipboard  
+![](http://images.thoughtbot.com/TIL/copyClone_s.jpg)
+* in Terminal (as are all `commands` that follow) `git clone <PASTE CLIPBOARD>`
```

Comment:
I am entertained, but do not disagree, with the assumption that people who don't know git use OS X.